sematic/tests/test_client.py

```diff
@@ -1,6 +0,0 @@
-# Sematic
-from sematic import client
-
-
-def test_client():
-    assert client.get_artifact_value
```

Comment:
Why remove this file?

Reply:
This was one of the mypy discoveries.
`client.get_artifact_value` is the `api_client.get_artifact_value_by_id` method, so the assert checks that a method is not falsey, and will never fail.
I assumed the test was either incorrect or incomplete.
Writing a correct test was out of the scope of this patch, so I removed the existing stub instead.

Reply:
I think the test was simply there to make sure that the function exists on `client`.
Can you recover it and maybe do this instead:
```python
try:
    client.get_artifact_value
except AttributeError as e:
    assert False, str(e)
```

Reply:
Done.